code_schemes/rqas/aik/aik_incidents_of_violence_and_polarisation.json

```diff
@@ -13,6 +13,86 @@
             "VisibleInCoda": true,
             "Shortcut": "e"
         },
+        {
```

Comment:
Does it make sense to sort these codes by yes, no, neutral, don't know?

Reply:
Sorted, thanks.